app.js

```diff
@@ -0,0 +1,92 @@
+const billAmount = document.querySelector(".bill-amount");
+const tips = document.querySelectorAll(".tip");
+const customTip = document.querySelector(".custom");
+const numberOfPeople = document.querySelector(".people-tip");
+const tipAmount = document.querySelector(".total-tip");
+const totalTip = document.querySelector(".total");
+const reset = document.querySelector("button");
+
+let totalBill = 0;
+let numberOfPersons = 1;
+let tip = 0;
+let tipsArray = Array.from(tips);
+
+const billValueHandler = (e) => {
+  totalBill = Number(e.target.value);
+  calculateTipAmount();
+};
+
+const numberOfPeopleHandler = (e) => {
+  numberOfPersons = Number(e.target.value);
+  calculateTipAmount();
+};
+
+const tipsHandler = (e) => {
+  tipsArray.forEach((element) => {
+    element.addEventListener("click", () => {
+      tipsArray.forEach((item) => {
+        if (item !== element) {
```

Comment:
I think here you can use `.filter(item => item !== element)` instead of the if statement as it would be more consistent with the functional programming approach and more readable.

Reply:
Done!